mono/metadata/debug-mono-ppdb.c

```diff
@@ -518,16 +518,39 @@ mono_ppdb_lookup_locals (MonoDebugMethodInfo *minfo)
 	scope_idx = start_scope_idx;
 	mono_metadata_decode_row (&tables [MONO_TABLE_LOCALSCOPE], scope_idx-1, cols, MONO_LOCALSCOPE_SIZE);
 	locals_idx = cols [MONO_LOCALSCOPE_VARIABLELIST];
-	while (scope_idx == tables [MONO_TABLE_LOCALSCOPE].rows) {
+
+	// https://github.com/dotnet/roslyn/blob/master/docs/specs/PortablePdb-Metadata.md#LocalScopeTable
```

Comment:
nit: link to the specific commit instead of master here, it could get moved around